CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+#### 0.29.0+1 - 2018-07-19
```

Comment:
This version has already been published.
https://pub.dartlang.org/packages/quiver/versions/0.29.0+1

Reply:
ack. Fixed.